src/VRCFaceTracking.Avalonia/Controls/SettingsExpander.axaml

```diff
@@ -4,23 +4,32 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
              xmlns:controls="clr-namespace:VRCFaceTracking.Avalonia.Controls"
+             xmlns:converters="clr-namespace:VRCFaceTracking.Avalonia.Converters"
              x:Class="VRCFaceTracking.Avalonia.Controls.SettingsExpander">
   
   <Expander IsExpanded="False"
             VerticalAlignment="Stretch"
             HorizontalContentAlignment="Stretch"
             HorizontalAlignment="Stretch">
     <Expander.Header>
-      <StackPanel HorizontalAlignment="Stretch" Margin="0,10,0,10">
-        <TextBlock Text="{Binding Title, RelativeSource={RelativeSource AncestorType=controls:SettingsExpander}, FallbackValue='Setting'}"
-               FontSize="16"
-               FontWeight="SemiBold"
-               Margin="5"/>
-        <TextBlock Text="{Binding Description, RelativeSource={RelativeSource AncestorType=controls:SettingsExpander}, FallbackValue='Setting Description'}"
-                   FontSize="14"
-                   Foreground="{DynamicResource SystemChromeGrayColor}"
-                   Margin="5"/>
-      </StackPanel>
+      <Grid ColumnDefinitions="Auto,Auto,*">
+        <PathIcon Data="{Binding IconPath, RelativeSource={RelativeSource AncestorType=controls:SettingsExpander}, Converter={x:Static converters:TypeConverters.IconConverter}}"
+                  Grid.Column="0"
+                  Background="#00000000"
```

Comment:
You should be able to remove a fully transparent background from a PathIcon, and it won't render.